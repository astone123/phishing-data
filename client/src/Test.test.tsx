import * as React from 'react';
import * as renderer from 'react-test-renderer';

const TestComponent: React.SFC<{}> = () => {
  return <div>This is a test component</div>;
};

test('Test component test', () => {
  const tree = renderer.create(<TestComponent />).toJSON();
  expect(tree).toMatchSnapshot();
});
