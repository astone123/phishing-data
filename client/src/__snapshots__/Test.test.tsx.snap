// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test component test 1`] = `
<div>
  This is a test component
</div>
`;
